coalib/output/ConsoleInteraction.py

```diff
@@ -308,7 +308,8 @@ def print_results_formatted(log_printer,
     default_format = ('id:{id}:origin:{origin}:file:{file}:line:{line}:'
                       'column:{column}:end_line:{end_line}:end_column:'
                       '{end_column}:severity:{severity}:severity_str:'
-                      '{severity_str}:message:{message}')
+                      '{severity_str}:message:{message}:message_arguments:'
+                      '{message_arguments}')
```

Comment:
message arguments should not be in the default_format, instead document it in our arparser